### ISSUE TITLE
Pool-validator produces 0 blocks being in the current_validators group.

### ISSUE THREAD

**Issue by manferber** [nearprotocol/stakewars#562]
Last week the network had some issues with RPC, my pool was constantly missing blocks and was kicked out. And while other pools were producing ~5-10% fewer blocks, my pool **joe1** was missing more blocks (15-20%). I tried to solve this issue by:

- changing the validator-key of my staking-pool to the one I generated.

Right after this attempt, the pool-validator started producing 0 blocks each time. 

What I've tried to solve the issue:
1) checked if the value of "account_id" in the validator_key.json is the same as my pool has. Followed the recommendations from the troubleshooting guide: https://github.com/nearprotocol/stakewars/blob/master/troubleshooting.md

(the name of the pool matched the account_id in validator_key.json)

2) 1.generated the new key by issuing the command `near generate-key` on my near-shell,
   2. then I replaced the existing public and secret keys located in the validator_key.json with the newly-generated key-pair.
   3. removed the data folder from /.near/betanet
   4. restarted the docker
   5. after full synchronization, I issued the command `update_staking_key` and updated the keys of my pool in accordance with the key located in the validator_key.json file.

( This procedure didn't help also, in 2 epochs the keys were identical, but the blocks weren't produced ) 

3) tried to compile the binaries locally, instead of running nearcore with docker. 
( this didn't help also)

4) checked the versions of betanet, the one on my machine was identical to the version of the network.

5) tried to move my validator to another server, didn't help as well.

6) ports are open as they were when my pool was successfully producing blocks. 

I know that the exact same issue is happening to, at least, one validator of betanet right now (**mmm** pool).  But there's a good chance that other validators with 0 blocks are dealing from the same issue as me. 

Here are the screenshots showing that both the **account_id** and **public_key** of my pool are identical to those mentioned in the validator_key.json:

<img width="1013" alt="Screenshot 2020-07-19 at 17 00 53" src="https://user-images.githubusercontent.com/64081717/87876550-77071980-c9e1-11ea-9eef-432d28f46529.png">

<img width="774" alt="Screenshot 2020-07-19 at 16 59 57" src="https://user-images.githubusercontent.com/64081717/87876552-79697380-c9e1-11ea-88f3-83700f3ece06.png">

**Comment by breeze158** [comment]
Yes, i have same issue, I have tried to move to a new vps,undeletate,withdraw, fully synced(on new vps),update public-key,delegate again. Now my validator produces 0 block being in the current_validators group.

**Comment by bowenwang1996** [comment]
Does migrating to a different pool help?

**Comment by manferber** [comment]
I haven't tried it yet - wanted to solve the issue with the existing pool before switching to another one. And would like to understand why this happened and what is the possible solution to this case.

Will migrate to another pool-contract if this will be the only option left.

**Comment by breeze158** [comment]
migrate to a new pool is ok(for me)

**Comment by bowenwang1996** [comment]
@manferber did you ping the pool after you change your validator key?

**Comment by manferber** [comment]
As I understand, my pool was automatically pinged when I executed the `update_staking_key` function. 

P.S. I tried to change the validator-keys to the last pair that was working, I had the backup of it. And this is the same key mentioned in the genesis.json.  Also didn't work, 0 blocks. 

Migrating to another pool

**Comment by bowenwang1996** [comment]
What I meant is whether you pinged your staking pool after you update your validator key. If you do not it will stop producing blocks and get kicked out because of the validator key change, unless you are carefully enough to reuse the old key for two epochs before the new key take effect.

**Comment by manferber** [comment]
I 100% done this procedure (ping after changing the key) a few times during the last week. And I know the person who faced the same issue, and he pinged the pool with new keys. Didn't help also.

**Comment by joesixpack** [comment]
Having same issue.  0 blocks produced.

**Comment by marat586l** [comment]
The name of my pool was mmm and, as indicated in this report, I had exactly the same problems and performed similar actions. Then I started a new node and tried again. Nothing helped, the produced blocks = 0! I.e. issues did not depend on the node in any way.
I had to deploy a new pool, now named mmm.stakehouse.test and only after that the block production is 100%!

**Comment by bowenwang1996** [comment]
Thanks for the report. I'll investigate.
